oop/tests/unit/helpers/validators.spec.js

```diff
@@ -0,0 +1,147 @@
+import * as Helper from "@/helpers/validators.js";
```

Comment:
Usually for named function exports we import them as:

```javascript
import {
  functionA,
  functionB
} from './file-name'
```

Reply:
Fixed in commit fa7b6f8c78d23f555c25f0e22cb2b910c9467c3f